tools/Repo-Tasks.psm1

```diff
@@ -293,3 +296,4 @@ export-modulemember -Function Get-BuildScopes
 export-modulemember -Function Start-Build
 export-modulemember -Function Invoke-CheckinTests
 export-modulemember -Function Install-VSProjectTemplates
+export-modulemember -Function Set-TestMode
```

Comment:
are you missing adding a function, seems you have exported the function, but I am not sure if that function is implemented.